src/helpers/leta.ts

```diff
@@ -0,0 +1,63 @@
+import { sendMessage } from 'webext-bridge/background';
+
+import { removeLetaCookies, setLetaCookies } from './cookies';
+import { checkFPI } from './fpi';
+
+export type MullvadAccount = string;
+export type DataAccount = {
+  account: MullvadAccount;
+};
+
+export const getMullvadAccount = async (): Promise<MullvadAccount> => {
+  const { mullvadAccount } = await browser.storage.local.get('mullvadAccount');
+  return mullvadAccount;
+};
+
+export const letaLogin = async (account: string) => {
+  try {
+    const requestData = {
+      method: 'POST',
+      body: JSON.stringify({
+        account_number: account,
+      }),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+
+    const response = await fetch('https://api.mullvad.net/auth/v1/webtoken', requestData);
```

Comment:
Be aware that fetch does not throw errors (like axios). Fetch only throws error when a network error occurs.
Instead you should check the boolean `response.ok`